cdap-tms/src/main/java/co/cask/cdap/messaging/MessagingUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.messaging;
+
+import co.cask.cdap.api.common.Bytes;
+import co.cask.cdap.proto.id.NamespaceId;
+import co.cask.cdap.proto.id.TopicId;
+
+/**
+ * Utility class for LevelDB table operations.
+ */
+public final class MessagingUtils {
+
+  private MessagingUtils() {
+    // prevent object creation
+  }
+
+  public static byte[] toBytes(TopicId topicId) {
+    String topic = topicId.getNamespace() + ":" + topicId.getTopic() + ":";
+    return Bytes.toBytes(topic);
+  }
+
+  public static TopicId toTopicId(byte[] topicBytes) {
+    String topic = Bytes.toString(topicBytes);
```

Comment:
Does this require `topicBytes` to be exact? Meaning we need to copy the row key into a separate `byte[]`? Better for this method to take `int offset, int length` so that it can decode from a sub-array.